graylog2-server/src/main/java/org/graylog2/web/resources/WebInterfaceAssetsResource.java

```diff
@@ -104,7 +109,36 @@ public FileSystem load(@Nonnull URI key) throws Exception {
                 });
     }
 
-    @Path("assets/plugin/{plugin}/{filename}")
+    @Path(ASSETS_PREFIX + "/" + FAVICON)
```

Comment:
Couldn't this prefix be placed on the @Path annotation on the class level, so that we do not have to concatenate the strings on each method?